graphqlapi/src/main/java/io/stargate/graphql/web/DropwizardServer.java

```diff
@@ -50,6 +51,9 @@
 
 public class DropwizardServer extends Application<Configuration> {
 
+  public static final String[] NON_API_URI_REGEX =
+      new String[] {"^/playground$", "^/graphql-schema$"};
```

Comment:
These could be combined into a single regexp, unless there is a specific reason for keeping them separate?
(combining would result in a single RE that is faster to evaluate than going over 2 separately)

Reply:
Yes of course, stupid me..